package.json.sample

```diff
@@ -1,4 +1,4 @@
-{
+.{
   "name": "magento2",
```

Comment:
Praktiski šī nav problēma, it īpaši, jo tas ir tikai `.sample` fails, bet cenšamies necommitot neko, kas nav vajadzīgs konkrēto izmaiņu ieviešanai